CHANGELOG.md

```diff
@@ -1,5 +1,123 @@
+- chore: Bump github.com/zclconf/go-cty from 1.14.2 to 1.14.3 [\#2016](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2016) ([dependabot[bot]](https://github.com/apps/dependabot))
+- chore: Run Go tests using specific packages [\#2014](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2014) ([lantoli](https://github.com/lantoli))
+- chore: Creates project for test execution [\#2010](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2010) ([lantoli](https://github.com/lantoli))
+- chore: Adjusts go.mod files after removal of integration tests [\#2008](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2008) ([AgustinBettati](https://github.com/AgustinBettati))
+- chore: Reuses projects in executions in some resources and rename mig tests [\#2007](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2007) ([lantoli](https://github.com/lantoli))
+- chore: Simplifies makefile [\#2003](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2003) ([lantoli](https://github.com/lantoli))
+- doc: Adds export example to mongodbatlas\_cloud\_backup\_snapshot\_export\_job resource doc [\#2002](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2002) ([zach-carr](https://github.com/zach-carr))
+- chore: Enables tests in CI for `mongodbatlas_privatelink_endpoint_service_data_federation_online_archive` resource [\#2001](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2001) ([lantoli](https://github.com/lantoli))
+- chore: Deletes outdated integration tests [\#1999](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1999) ([lantoli](https://github.com/lantoli))
+- chore: Unifies pass of org variables in tests [\#1997](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1997) ([lantoli](https://github.com/lantoli))
+- chore: Stops using Network org in tests [\#1996](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1996) ([lantoli](https://github.com/lantoli))
+- chore: Changes skip project logic for cleanup action [\#1995](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1995) ([lantoli](https://github.com/lantoli))
+- chore: Bump dorny/paths-filter from 3.0.1 to 3.0.2 [\#1994](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1994) ([dependabot[bot]](https://github.com/apps/dependabot))
+- chore: Bump tj-actions/verify-changed-files from cb704d8c717959cf92ec9db9131372bc20356aa2 to 79a8cef5d9ef3ab541ee07ef179bd6c3c2d42ecc [\#1993](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1993) ([dependabot[bot]](https://github.com/apps/dependabot))
+- chore: Bump peter-evans/create-pull-request from 6.0.0 to 6.0.1 [\#1992](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1992) ([dependabot[bot]](https://github.com/apps/dependabot))
```

Comment:
suggest to move all bumps together at the end of the section

Reply:
done